Makefile

```diff
@@ -478,6 +482,13 @@ generate-addons: fetch-calico-manifests ## Generate metric-server, calico calico
 	$(KUSTOMIZE) build $(ADDONS_DIR)/calico-ipv6 > $(ADDONS_DIR)/calico-ipv6.yaml
 	$(KUSTOMIZE) build $(ADDONS_DIR)/calico-dual-stack > $(ADDONS_DIR)/calico-dual-stack.yaml
 
+.PHONY: generate-aso-crds
+generate-aso-crds: $(YQ)
+	curl -fSsL "https://github.com/Azure/azure-service-operator/releases/download/$(ASO_VERSION)/azureserviceoperator_customresourcedefinitions_$(ASO_VERSION).yaml" | \
+		$(YQ) e '. | select($(foreach name,$(ASO_CRDS),.metadata.name == "$(name)" or )false)' - | \
+		sed 's/\$$\$$/$$$$$$$$/g' \
```

Comment:
lol

Reply:
Also this is slightly anticipatory for once we start using any of the following CRDS:
```
managedclusters.containerservice.azure.com
virtualmachines.compute.azure.com
virtualmachinescalesets.compute.azure.com
```